package.json

```diff
@@ -42,12 +42,6 @@
     "lodash.partition": "4.6.0"
   },
   "ava": {
-    "timeout": "30s",
-    "environmentVariables": {
```

Comment:
Currently, I don't know how we can make package.json "ava" property make understand that env vars in node.js.yml file should take precedence.

Reply:
Having a .env and not relying on ava should fix this.

Reply:
agreed, let's handle in this issue: https://github.com/neume-network/strategies/issues/94